### ISSUE TITLE
Can't run the server, Unexpected Token error after trying to run the server with node SRC

### ISSUE THREAD

**Issue by Ncarauj012** [OpenAsar/arrpc#119]
Running Linux Mint 21.3 Cinnamon version 6.0.4

Trying to run "node src" after going through the server setup only results in: 
file:///home/nc/arrpc/src/server.js:78
        socket.send?.({
                    ^

SyntaxError: Unexpected token '.'
    at Loader.moduleStrategy (internal/modules/esm/translators.js:133:18)

did i skip a step? am i missing a dependency or something?

**Comment by MaxTheDhole** [comment]
Came across the same error, turns out my node version was below the prerequisites for arrpc, check that out just in case

**Comment by Ncarauj012** [comment]
Yup that solved it, maybe put that on the main page as a warning
